sql/updates/world/2015_06_08_10_world.sql

```diff
@@ -0,0 +1,14 @@
+-- the spell1 column was missused for this NPC. This NPC is undead. He cannot be MCed. And the spell 29307 should be casted using SmartAI.
+UPDATE `world`.`creature_template` SET `spell1`='0' WHERE `entry`='16360';
+UPDATE `world`.`creature_template` SET `spell1`='0' WHERE `entry`='30303';
+
+-- The zombie chows cast on themselves the spell 29307 which makes their AA apply one stack of the debuff "Infected Wound".
+DELETE FROM `world`.`smart_scripts` WHERE `entryorguid`='16360';
+INSERT INTO `world`.`smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES ('16360', '0', '0', '0', '4', '0', '100', '31', '0', '0', '0', '0', '11', '29307', '0', '0', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0', '0', 'Zombie Chow - On Creature Aggro - Cast \' Infected Wound\' on self (No Repeat)');
+UPDATE `world`.`creature_template` SET `AIName`='SmartAI' WHERE `entry`='16360';
```

Comment:
@tkrokli: What joshchiwald means is that this spell cast can and should be moved to `creature_template_addon.auras` since it is an aura cast on reset/spawn.